packages/kit/src/core/build/index.js

```diff
@@ -281,7 +281,8 @@ async function build_server(
 									type: 'page',
 									pattern: ${route.pattern},
 									params: ${params},
-									parts: [${route.parts.map(file => s(file)).join(', ')}]
+									a: [${route.a.map(file => file && s(file)).join(', ')}],
```

Comment:
are there a more descriptive names we can use than `a` and `b`?

Reply:
plan `a` is we render `[layout0, layout1, ...layoutn, leaf]`, plan `b` is we render `[layout0, ...layoutn, error]`

Reply:
(plan c is we render the fallback error page — `[layout0, error0]` — and plan d is we return a bare-bones 500 or throw)

Reply:
Maybe we should call these `planA` and `planB`? Or perhaps better would be `pageStack` and `errorStack`?

Reply:
added a note to the declaration in `types.internal`